doc/user/content/concepts/sources.md

```diff
@@ -43,37 +43,18 @@ Component      | Use
 
 Materialize bundles native connectors for the following external systems:
 
-- [Kafka](/sql/create-source/kafka)
-- [Redpanda](/sql/create-source/kafka)
-- [PostgreSQL](/sql/create-source/postgres)
-- [MySQL](/sql/create-source/mysql/)
-- [SQL Server](/sql/create-source/sql-server)
-- [Webhooks](/sql/create-source/webhook/)
+{{< include-md file="shared-content/multilink-box-native-connectors.md" >}}
 
 For details on the syntax, supported formats and features of each connector, check out the dedicated `CREATE SOURCE` documentation pages.
 
-### Formats
 
-Materialize can decode incoming bytes of data from several formats:
+## Sources and clusters
```

Comment:
https://preview.materialize.com/materialize/32903/concepts/sources/#sources-and-clusters